src/job-feed/JobFeedPage.js

```diff
@@ -23,11 +24,14 @@ const useStyles = makeStyles({
   },
 });
 
-function JobFeedPage({ errorAutoHide }) {
+function JobFeedPage({ errorAutoHide = 2500 }) {
+  const history = useHistory();
   const classes = useStyles();
   const { data, loading, error } = useQuery(GET_JOBS);
   const [showError, setShowError] = useState(false);
 
+  const memoJobs = React.useMemo(() => (data ? data.jobs.map((it) => <Job key={it.id} id={it.id} name={it.name} description={it.description} isAvailable={it.isAvailable} companyName={it.company.name} updatedAt={it.updatedAt} onClick={() => history.push(`/jobs/${it.id}`)} />) : <p>There are no jobs</p>), [data]);
```

Comment:
Control your line lengths, this is insane :rofl: 
Why not pass the `job` object to `Job` instead of passing each prop separately?
No point in passing an `onClick` handler either, it can be defined within `Job`